### DISCUSSION TITLE
Yeni site görünümünde cüzdan bağlama görünmüyor.

### DISCUSSION THREAD

**Discussion by ZiyaGungor** [IBAX-io/go-ibax#625] [Soru Cevap - Türkçe] [answered]
İbax yeni site görüntüsü güzel olmuş ancak cüzdan bağlamak nereden yapılıyor.

**Comment by Yusufknk** [comment, marked as answer]
My dashboard-Get İbxc aşamalarını takip edin. İlk başta cüzdan adresi bağlama seçeneği çıkacak, ama daha önce bağladıysanız lütfen o esnada biraz bekleyin. Sistemin cüzdanınızı tanımlamış olarak göreceksiniz.

**Reply by ZiyaGungor** [reply]
Biraz karmaşık olmuş ama buldum.

**Reply by ZiyaGungor** [reply]
Githup puanları neden yansımıyor hesaba o konuda eksik yaptığımız bir nokta mi var

**Reply by volvoxx8** [reply]
Daha önce tanımlayanlar başka bir cüzdan tanımlamak isterse ?
